docs/quickstarts/ratify-with-aws-signer.md

```diff
@@ -202,7 +202,7 @@ curl -sSLO https://d2hvyiie56hcat.cloudfront.net/aws-signer-notation-root.cert
 helm install ratify \
     ratify/ratify --atomic \
     --namespace gatekeeper-system \
-    --set-file notationCert=./aws-signer-notation-root.cert \
+    --set-file notationCerts={./aws-signer-notation-root.cert} \
```

Comment:
just want to confirm that the file path does not need to be in quotes?

Reply:
The helm [spec ](https://helm.sh/docs/helm/helm_template/) states:
`set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2)`